builds/misc/templates/build-api-proxy.yaml

```diff
@@ -0,0 +1,24 @@
+parameters:
+  name: 'API proxy'
+  imageName: 'api-proxy-module'
+  namespace: 'microsoft'
```

Comment:
what is namespace used for?

Reply:
nit: seems it should be defined as default in buildAPIProxy.sh.

Reply:
It is used by the build script. Eventually this is what it creates: 
![image](https://user-images.githubusercontent.com/33438817/93150282-88049c00-f6ad-11ea-99eb-fd2151598b1b.png)

Reply:
buildAPIProxy.sh is actually very similar to buildimage.sh. I want to keep them the same as much as possible. I believe there is a lot of work to be done for the pipeline. Merging the two is one of them.